ioc/Jail.py

```diff
@@ -1602,6 +1602,11 @@ def devfs_ruleset(self) -> ioc.DevfsRules.DevfsRuleset:
         if self._allow_mount_zfs == "1":
             devfs_ruleset.append("add path zfs unhide")
 
+        if self.config["allow_vmm"] is True:
+            devfs_ruleset.append("add path vmm unhide")
+            devfs_ruleset.append("add path vmm/* unhide")
```

Comment:
as discussed elsewhere, it would be good to create a *concrete* rule per vmm container, rather than exposing all these devices to each jail